src/index.html

```diff
@@ -12,7 +12,46 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="page__product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="page__product-card-img"
+      />
+      <div class="page__product-card-info">
+        <h1 class="page__product-card-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the page, so you need to use the another tag here
```suggestion
        <h2 class="page__product-card-title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```